internal/querynodev2/segments/metricsutil/observer.go

```diff
@@ -179,12 +167,6 @@ type promMetricsObserver struct {
 	nodeID string
 	label  SegmentLabel // never updates
 
-	DiskCacheLoadTotal                   prometheus.Counter
```

Comment:
why we remove these metrics?

Reply:
There's already exported in `cache monitor.go`
repeated metrics can be removed.